lists/ch/ch-bs-cr.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": {
+    "en": "Official Commercial Register of the Canton of Basel-Stadt",
+    "local": "Offizielle Handelsregisterinformationen des Kantons Basel-Stadt"
+  },
+  "url": "https://bs.chregister.ch/cr-portal/suche/suche.xhtml",
+  "description": {
+    "en": "The purpose of the commercial register is to record and disclose the business-relevant facts of the more than 17,200 companies, foundations and associations based in the canton of Basel-Stadt, thereby contributing to transparency and ensuring legal certainty in business transactions.\nThis portal allows you to find the published data of all companies in our department. This includes in particular the legal form, purpose and registered persons. The data can be displayed or downloaded directly in the browser or as a PDF file. Please note that extracts are made available on this portal without guarantee and are not legally binding. However, you have the option of ordering commercial register extracts which are certified by us and therefore legally binding. To do this, please use the order form."
+  },
+  "coverage": [
+    "CH"
+  ],
+  "subnationalCoverage": [
+    "CH-BS"
+  ],
+  "structure": [
+    "company"
+  ],
+  "sector": [],
+  "code": "CH-BS",
```

Comment:
Please change this to CH-BS-CR. (I've already edited the filename and directory to match this.)